docs/project-setup/filesystem-sync.md

```diff
@@ -1,41 +1,22 @@
 # Filesystem Sync
 
-The default NFS filesystems that Outrigger sets up provides easy sharing of code and files with your project containers. 
-However, NFS can be slow when writing, reading, or scanning thousands of files when compared to native filesystem performance. 
-Some modern tool kits and package managers favor large numbers of small files and libraries so maintaining high performance 
-of a build becomes challenging when your code base is on a volume mounted NFS share.
+Outrigger uses [Unison](https://github.com/bcpierce00/unison) to provide high-performance, bi-directional file synchronization between your local operating system and Docker containers.
 
-NFS also has the downside of not propagating filesystem modification events. This can be problematic if you want to run
-a process in your container to rebuild or update compiled project assets when modification notifications are triggered
-due to local file editing in your IDE.
+From the root of your project, all the files are synced via a dedicated container and exposed as a named [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/) to the rest of your application. This volume is used as the file "source" for a volume mount to allow nearly native filesystem performance and features.
 
-In order to better support these types of operations, Outrigger provides support for multi-directional filesystem syncing 
-(which includes filesystem notifications) using `unison`.  Your project root folder is synced via `unison` into a sync 
-container and exposed as a Docker Volume to the rest of your application. This volume is then used as your mount point 
-instead of the NFS share and allows nearly native filesystem performance and features.
+This is not used as the only means of sharing code or data into the filesystem, but it is the recommended approach for sharing primary application code that must also be readily available to developers and their local tools (e.g., IDEs).
```

Comment:
I'd reword the first sentence to the following and move this as the end of the first paragraph.

"This is not the only means of sharing code or data with application containers, but it is the recommended . . . "